pkg/v12/resource/tiller/resource.go

```diff
@@ -83,7 +83,13 @@ func (r *Resource) ensureTillerInstalled(ctx context.Context, clusterGuestConfig
 		return microerror.Mask(err)
 	}
 
-	err = tenantHelmClient.EnsureTillerInstalled(ctx)
+	values := []string{
+		"spec.template.spec.priorityClassName=giantswarm-critical",
+		"spec.template.spec.tolerations[0].effect=NoSchedule",
+		"spec.template.spec.tolerations[0].key=node-role.kubernetes.io/master",
+		"spec.template.spec.tolerations[0].operator=Exists",
+	}
+	err = tenantHelmClient.EnsureTillerInstalledWithValues(ctx, values)
```

Comment:
This basically adds the master toleration and priority class to tiller, here are some snippets from `kubectl get pod tiller -o yaml`:
```
  priority: 1000000000
  priorityClassName: giantswarm-critical
...
  tolerations:
  - effect: NoSchedule
    key: node-role.kubernetes.io/master
```

Reply:
This is cool but v12 is in active releases. Shouldn't it just be in v13?

Reply:
I was not sure if we needed to backport it, so I did so to be safe. The problem is that a cluster with 1 worker can get deadlocked right now - so I thought it might be nice to backport?

Not sure - I am fine with removing this code and just have it in V13.

Reply:
I think we should add this to the WIP release like usual. If this is causing problems in production it can be backported as a separate PR.

Reply:
Sounds like a decent idea.

Reply:
Done.